portfolio/src/main/webapp/CommentHandler.js

```diff
@@ -1,10 +1,10 @@
 /** The class in charge of comment loading. */
-export default class CommentHandler {
+class CommentHandler {
   /**
    * Loads all comments from backend.
    * This function is called when the page is loaded.
-   * @param {HTML div object} commentBox The div in document with id "comment-box".
-   * @param {HTML template} commentCell The template for one comment.
+   * @param {!HTML div object} commentBox The div in document with id "comment-box".
+   * @param {!HTML template} commentCell The template for one comment.
```

Comment:
As a note, the objects are supposed to be actual types. It doesn't matter for this code, but if you use the Closure compiler, it's going to complain and throw a build error.

Reply:
I'll call a template in soy :). And thanks for the reminder.